packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewModalContent.tsx

```diff
@@ -60,25 +63,28 @@ export const TransactionReviewModalContent = ({
     const { account } = selectedAccount;
     const { precomposedTx, serializedTx } = txInfoState;
 
+    const precomposedForm: FormState | StakeFormState | undefined = isStakeState(txInfoState)
+        ? txInfoState.precomposedForm
+        : txInfoState.drafts[account?.key ?? ''];
+
+    const decreaseOutputId = precomposedTx?.useNativeRbf
+        ? precomposedForm?.setMaxOutputId
+        : undefined;
```

Comment:
The thing is that this cannot be moved to selector in context of this file because of the way how is this component designed. The whole state of this module is served a as a component prop (yes, you heard me right 😩) and it can be either `SendState` or `StakeState`. These states are very similar, but not the same 😤. So if I would create a selector in `SendReducer` that we use also in mobile, it would broke the desktop send form for staking. 

I agree that it is not ideal, but I'm sure I don't want to refactor this as part of this PR. Because it would be huge refactor across many packages of desktop suite.

Reply:
I thought that's what the `isStakeState` is for and you could use it in the selector. Could you elaborate?

Reply:
Special selector created here: [05f5b4b](https://github.com/trezor/trezor-suite/pull/12968/commits/05f5b4bafada99e14c5cecad39deaa76eab5f064)

Not much optimistic that this solved anything, but the code looks a bit better at least.